### ISSUE TITLE
Create new atm process to read in gas optics

### ISSUE THREAD

**Issue by PeterCaldwell** [E3SM-Project/scream#841]
Radiation expects profiles for 
```
active_gases = (/ &
      'H2O', 'CO2', 'O3 ', 'N2O', &
      'CO ', 'CH4', 'O2 ', 'N2 ' ).
```
co2, ch4, n2o are namelist variables assumed to be spatially invariant in EAM. Prescribed O3 should be read in.  H2O profiles should come from qv as used by the rest of the model. I'm not sure where CO should come from. 

We should create a new atm process (which is really just a stub for future chemistry work) which just reads in and creates these prescribed fields. I don't think this is needed for v0 since it already accomplishes this... somehow. Just a C++ exercise.

**Comment by bartgol** [comment]
Isn't that what ScorpioInput does? @AaronDonahue

**Comment by PeterCaldwell** [comment]
@bartgol - I think the task here is to use ScorpioInput to read in these specific files and then to process them appropriately. ScorpionInput makes that task easy but doesn't actually do the specific work for us...

**Comment by bartgol** [comment]
Oh, so the task is "make rad use ScorpioInput to load gas optics"?

**Comment by PeterCaldwell** [comment]
> Oh, so the task is "make rad use ScorpioInput to load gas optics"?

Yes, just use ScorpioInput to load gas optics. These ScorpioInput calls should be in their own atm process though for future extensibility.
